rostwitter/python/rostwitter/twitter.py

```diff
@@ -1,16 +1,17 @@
 # originally from https://raw.githubusercontent.com/bear/python-twitter/v1.1/twitter.py  # NOQA
 
+import math
 import json as simplejson
 import requests
+from itertools import zip_longest
```

Comment:
I think this line may be only for python3.
Something like...
```suggestion
try:
    from itertools import zip_longest
except ImportError:
    from itertools import izip_longest
```

Reply:
Thanks! I fixed it.